pkgs/by-name/i3/i3bar-river/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, glib
+, pango
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "i3bar-river";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "MaxVerevkin";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-c5R5V5J1ETBl6JAdNDSxa94OeMyqbTAUmJHJCo1B+WQ=";
+  };
+
+  cargoHash = "sha256-D/WKv8rhb/ZGuVEZDp83PZxJydHbnZUQp+kVNlMBUCs=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ glib pango ];
+
+  meta = with lib; {
+    description = "A port of i3bar for river";
+    homepage = "https://github.com/MaxVerevkin/i3bar-river";
+    license = licenses.gpl3Only;
+    mainProgram = "i3bar-river";
+    platforms = platforms.unix;
```

```suggestion
    platforms = platforms.linux;
```

Does not build on darwin, hasn't tested BSD.